jobs/transforms/exp_validation.py

```diff
@@ -45,3 +46,13 @@ def _get_change_domain_class(self, input_model): # pylint: disable=unused-argume
             changes made by commit commands of the model.
         """
         return exp_domain.ExplorationChange
+
+
+@validation_decorators.RelationshipsOf(exp_models.ExplorationModel)
+def exploration_model_relationships(model):
+    """Yields how the properties of the model relates to the ID of others."""
+
+    exploration_model_ids = [
+        'exploration-%s-%s' % (model.id, version)
+        for version in python_utils.RANGE(1, model.version + 1)]
```

Comment:
Ah, you cannot use the properties of a model from this function. The reason is, `model` is a **class**, _not_ a model instance.

Thanks for flagging this -- this is an edge case I hadn't considered. FYI, this will probably need a special decorator, so I think you should skip it for now. I'm really sorry this gave you so much trouble, I'll try to make the documentation clearer so this never happens again

Reply:
Thanks for the clarification. Implementing the simples one's for now then.